client/web/reporter/src/components/Report/Blocks/DisplayElements/Configurators/Chart.vue

```diff
@@ -160,16 +160,32 @@
               {{ column.label }}
             </h6>
 
-            <b-form-group
-              :label="$t('chart.configurator.metrics.stack.name')"
-              :description="$t('chart.configurator.metrics.stack.description')"
-              label-class="text-primary"
-              class="mb-0"
-            >
-              <b-form-input
-                v-model="column.stack"
-              />
-            </b-form-group>
+            <b-row>
+              <b-col>
```

Comment:
Lets make it responsive, take a look at other forms like this (md 6 i think cols 12)